lifetimes/plotting.py

```diff
@@ -172,6 +174,48 @@ def plot_expected_repeat_purchases(model, **kwargs):
     return ax
 
 
+def plot_history_alive(model, transactions, datetime_col, units, freq='D', **kwargs):
+
+    """
+    Draws a graph showing the probablility of being alive for a customer in time
+    :param model: A fitted lifetimes model
+    :param transactions: a Pandas DataFrame containing the transactions history of the customer_id
+    :param datetime_col: the column in the transactions that denotes the datetime the purchase was made
+    :param units: the number of time units since the birth we want to draw the p_alive
+    :param freq: Default 'D' for days. Other examples= 'W' for weekly
+    """
+
+    from matplotlib import pyplot as plt
+
+    start_date = kwargs.pop('start_date', min(transactions[datetime_col]))
+    ax = kwargs.pop('ax', None) or plt.subplot(111)
+
+    # Get purchasing history of user
+    customer_history = pd.DataFrame(transactions[[datetime_col]].copy())
```

Comment:
The `pd.DataFrame` is not needed as `transactions[[datetime_col]]` is a dataframe